pkg/k8shandler/curation_test.go

```diff
@@ -29,9 +30,6 @@ func TestNewCuratorCronJobWhenFieldsAreUndefined(t *testing.T) {
 	if resources.Requests[v1.ResourceCPU] != defaultFluentdCpuRequest {
 		t.Errorf("Exp. the default CPU request to be %v", defaultCuratorCpuRequest)
 	}
-	if podSpec.NodeSelector != nil {
```

Comment:
instead of removing this, we should expect that it would not be nil... can you make this report an error if `podSpec.NodeSelector == nil` ?

Reply:
Does it count that we have such test here? https://github.com/openshift/cluster-logging-operator/pull/191/files#diff-c61d3b9279a00c2a95db3179686043d7R21
Or you would like me to add such test for curation and visualization as well?

Reply:
I suppose it is redundant, I figured it would just be good to ensure that when the curator (and kibana) pods are created they still have the node selector (to prevent regression within those individual functions)